Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/audit/transform/AuditTransformsTest.java

```diff
@@ -89,7 +88,11 @@ protected void testGetEventIdentificationType(LogEventRequestType request, Strin
         }
         assertEquals(AuditTransformsConstants.EVENT_OUTCOME_INDICATOR_SUCCESS.toString(), eventIdentificationType
             .getEventOutcomeIndicator().toString());
-        assertEquals(getAuditTransforms().getServiceEventIdCode(), eventIdentificationType.getEventID().getCode());
+        if (isRequesting) {
+            assertEquals(getAuditTransforms().getServiceEventIdCodeRequestor(), eventIdentificationType.getEventID().getCode());
+        } else {
+            assertEquals(getAuditTransforms().getServiceEventIdCodeResponder(), eventIdentificationType.getEventID().getCode());
+        }
```

Comment:
If you'll already be pushing up another commit today, make sure these lines above are wrapped.